.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: main
+on:
+  - push
+  - pull_request
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby-version:
+          - 2.5
+          - 2.6
+          - 2.7
+          - 3.0
+        gemfile:
+          - rails_52.gemfile
+          - rails_60.gemfile
+          - rails_61.gemfile
+        exclude:
+          - ruby-version: [ 2.6, 2.7, 3.0 ]
+            gemfile: rails_52.gemfile
+          - ruby-version: 3.0
+            gemfile: rails_60.gemfile
+    steps:
+      - name: Install packages required for `curb` gem
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libcurl4 libcurl3-gnutls libcurl4-openssl-dev
```

Comment:
FYI: I don't known why but `curb` gem can't be compiled in GitHub Actions CI without these packages.

Reply:
:+1: